helm/hpcc/values.yaml

```diff
@@ -364,8 +366,9 @@ dafilesrv:
     servicePort: 7600
     visibility: local
 
- # Enabling this service exposes a dafilesrv and Dali for foreign file access for backward compatibility.
- # If enabled, both services should be secured with strict ingress rules.
+# Enable if bare-metal systems require read access to this systems' data planes via ~remote::
+# If legacy ~foreign:: access is required, Dali will also need to be exposed via a service definition in the dali configuration
+# NB: ingress rules should be added to limit access.
```

Comment:
NB: these comments are somewhat incidental to this PR, but thought worth adding to clarify